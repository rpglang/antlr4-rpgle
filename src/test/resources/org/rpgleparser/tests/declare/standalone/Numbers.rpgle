        DCL-S index INT(10);
        DCL-S count UNS(20);
        DCL-S variance FLOAT(8);