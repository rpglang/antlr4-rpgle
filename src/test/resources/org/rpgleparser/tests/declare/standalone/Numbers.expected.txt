/*===TOKENS===*/
DS_Standalone            <DCL-S>
ID                       <index>
INT                      <INT>
'('                      <(>
NUMBER                   <10>
')'                      <)>
FREE_SEMI                <;>
DS_Standalone            <DCL-S>
ID                       <count>
UNS                      <UNS>
'('                      <(>
NUMBER                   <20>
')'                      <)>
FREE_SEMI                <;>
DS_Standalone            <DCL-S>
ID                       <variance>
FLOAT                    <FLOAT>
'('                      <(>
NUMBER                   <8>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword index)))
      (datatype (datatypeName INT) (args ( (expression (number 10)) )))
      ;
    )
  )
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword count)))
      (datatype (datatypeName UNS) (args ( (expression (number 20)) )))
      ;
    )
  )
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword variance)))
      (datatype (datatypeName FLOAT) (args ( (expression (number 8)) )))
      ;
    )
  )
)
/*======*/